doc/serial.md

```diff
@@ -42,3 +43,172 @@ sudo systemctl disable nvgetty
 
 Followed by a reboot.
 
+# New serial protocol
+
+## Configuration
+
+Board A : UART 7
+
+Jetson : UART 2 (pins 8 (TX) and 10 (RX))
+
+230400-8N1 - Little endian
+
+8 bits 230400 baud 1 stop bit & no parity bit
+
+## Protocol
+
+General message layout
+
+|   SOF  | `cmd_id`  | `data_len` | `data ` |
+|--------|-----------|------------|---------|
+| 1 byte |  1 byte  |  1 byte    |         |
+| Fixed value : `0xFC` |      |   Length of `data`, in bytes |       |
+
+`cmd_id` possible values : 
+
+CS -> CV
+
+| Code    |     Description      |  `data_len` |
+|---------|----------------------|-------------|
+| `0x01`  | Game status          | 14          |
+| `0x02`  | Event (taproot: gamestage)   | 2        |
+| `0x03`  | Turret feedback      | 4            |
+| `0x04`  | Position feedback    | 22           |
+
+CV -> CS
+
+| Code    |     Description      |  `data_len` |
+|---------|----------------------|-------------|
+| `0x10`  | Turret target        | 4           |
+| `0x11`  | Movements            | 6           |
+| `0x12`  | Shoot order          | 0 (no data) |
+
+## Commands
+
+### Game status
```

Comment:
3v3 can also have 2 STD in place of a HRO or STY (if I remember correctly), is it accounted for?